### PR TITLE
[ACS-6688] excluding & fixing failing tests

### PR DESCRIPTION
**Pull request by datguychen**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [x] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Tests fail because there is an issue on the BE and tests are not excluded.

**What is the new behaviour?**
Excluding tests because of this issue: https://alfresco.atlassian.net/browse/ACS-6688

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: e2e/playwright/library-actions/exclude.tests.json [hunk 1]
```diff
@@ -1 +1,3 @@
-{}
+{
+    "C286326": "Recent files and Shared tabs are giving error 500 https://alfresco.atlassian.net/browse/ACS-6688"
```
Comment (AleksanderSklorz) [inline e2e/playwright/library-actions/exclude.tests.json:2]: @datguychen it's enough to have just link to jira, no need for title of task, could you please change for every test?

**Review by MichalKinas — APPROVED** [review]

File: e2e/playwright/library-actions/exclude.tests.json (hunk 1 above)
Comment (datguychen) [reply e2e/playwright/library-actions/exclude.tests.json]: Fixed

**Review by AleksanderSklorz — APPROVED** [review]

File: e2e/playwright/library-actions/exclude.tests.json (hunk 1 above)
Comment (AleksanderSklorz) [reply e2e/playwright/library-actions/exclude.tests.json]: thank you

File: projects/aca-playwright-shared/src/utils/timeouts.ts [hunk 2]
```diff
@@ -33,6 +33,7 @@ export const timeouts = {
   extraLarge: 20 * 1000,
   globalTest: 45 * 1000,
   extendedTest: 150 * 1000,
+  extendedLongTest: 200 * 1000,
   webServer: 240 * 1000,
-  globalSpec: 60 * 10 * 1000
+  globalSpec: 60 * 10 * 1000,
```
Comment (AleksanderSklorz) [inline projects/aca-playwright-shared/src/utils/timeouts.ts:38]: redundant comma at the end

File: projects/aca-playwright-shared/src/page-objects/pages/login.page.ts [hunk 3]
```diff
@@ -66,4 +66,10 @@ export class LoginPage extends BasePage {
     const type = await this.password.getAttribute('type');
     return type === 'text';
   }
+
+  async verifyUserLogin(): Promise<void> {
+    if(this.username.isVisible()){
+      await this.page.reload({waitUntil:"load"});
```
Comment (AleksanderSklorz) [inline projects/aca-playwright-shared/src/page-objects/pages/login.page.ts:72]: can you add space after { and before } ?

File: projects/aca-playwright-shared/src/page-objects/pages/login.page.ts [hunk 4]
```diff
@@ -66,4 +66,10 @@ export class LoginPage extends BasePage {
     const type = await this.password.getAttribute('type');
     return type === 'text';
   }
+
+  async verifyUserLogin(): Promise<void> {
+    if(this.username.isVisible()){
```
Comment (AleksanderSklorz) [inline projects/aca-playwright-shared/src/page-objects/pages/login.page.ts:71]: missing space after if and before {

File: projects/aca-playwright-shared/src/base-config/global-variables.ts [hunk 5]
```diff
@@ -33,5 +33,9 @@ export const users = {
   admin: {
     username: env.ADMIN_EMAIL,
     password: env.ADMIN_PASSWORD
+  },
+  contentIdentity: {
+    username: env.CONTENT_IDENTITY_USERNAME,
```
[the remark this answers is not in the document]
Comment (datguychen) [reply projects/aca-playwright-shared/src/base-config/global-variables.ts]: I forgot to delete that change in the code. Its deleted now

File: projects/aca-playwright-shared/src/page-objects/pages/login.page.ts (hunk 3 above)
Comment (datguychen) [reply projects/aca-playwright-shared/src/page-objects/pages/login.page.ts]: added

File: projects/aca-playwright-shared/src/page-objects/pages/login.page.ts (hunk 4 above)
Comment (datguychen) [reply projects/aca-playwright-shared/src/page-objects/pages/login.page.ts]: added

File: projects/aca-playwright-shared/src/utils/timeouts.ts (hunk 2 above)
Comment (datguychen) [reply projects/aca-playwright-shared/src/utils/timeouts.ts]: deleted

**Review by AleksanderSklorz — APPROVED** [review]

**Pull request merged**